e2e/e2e_test.go

```diff
@@ -223,31 +442,23 @@ func verifyTransferFromValidatorAPI(setupEnv *setup.Setup, txResponse hedera.Tra
 	return transactionData, tokenAddress
 }
 
-func verifyDatabaseRecords(dbValidation *database.Service, expectedRecord *entity.Transfer, mintAmount string, signatures []string, t *testing.T) {
-	exist, err := dbValidation.VerifyDatabaseRecords(expectedRecord, mintAmount, signatures)
+func verifyBurnEventRecord(dbValidation *database.Service, expectedRecord *entity.BurnEvent, t *testing.T) {
+	exist, err := dbValidation.VerifyBurnRecord(expectedRecord)
 	if err != nil {
-		t.Fatalf("[%s] - Verification of database records failed - Error: [%s].", expectedRecord.TransactionID, err)
+		t.Fatalf("[%s] - Verification of database records failed - Error: [%s].", expectedRecord.Id, err)
 	}
 	if !exist {
-		t.Fatalf("[%s] - Database does not contain expected records", expectedRecord.TransactionID)
+		t.Fatalf("[%s] - Database does not contain expected records", expectedRecord.Id)
 	}
 }
 
-func prepareExpectedTransfer(routerContract *routerContract.Router, transactionID hedera.TransactionID, nativeAsset, amount, receiver string, statuses database.ExpectedStatuses, t *testing.T) *entity.Transfer {
-	expectedTxId := hederahelper.FromHederaTransactionID(&transactionID)
-
-	wrappedAsset, err := setup.WrappedAsset(routerContract, nativeAsset)
+func verifyTransferRecordAndSignatures(dbValidation *database.Service, expectedRecord *entity.Transfer, mintAmount string, signatures []string, t *testing.T) {
+	exist, err := dbValidation.VerifyDatabaseRecords(expectedRecord, mintAmount, signatures)
```

Comment:
nit: Rename the `VerifyDatabaseRecords` function to something more concrete, now that we have `VerifyBurnRecord`. It would seem that we are validating all records but actually only transfer and signature are validated